tests/installation/installer_desktopselection.pm

```diff
@@ -44,7 +44,9 @@ sub run() {
     assert_screen "$d-selected";
     if (check_var('VERSION', 'Tumbleweed')) {
         send_key 'alt-o';                                                      # configure online repos
+        wait_still_screen 3;                                                   # wait for the potential 'low memory warning' to show up
         assert_screen 'repo-list';
+        wait_screen_change { send_key $cmd{ok} } if match_has_tag 'repo-list-low_memory_warning';
```

Comment:
This is what I am doing 2 lines above with the `wait_still_screen`, wait for the (potential) dialog to pop up. Also, I replaced the existing needle so that it's explicit that either there is no popup (blank space where the popup would be) or there *is* a popup. I can think of one alternative which is wasting time by waiting with `check_screen` and just going on if there is no popup.